sensors/Makefile

```diff
@@ -11,7 +11,7 @@ include $(static-lib.mk)
 # Sensors SPI interface
 NAME := libsensors-spi
 LOCAL_HEADERS := sensors-spi.h
-LOCAL_HEADERS_DIR := nothing  # do not reinstall libsensors.h
+LOCAL_HEADERS_DIR := nothing  # do not reinstall libsensors/client.h
```

Comment:
also `sensors.h` - so the comment is still incorrect.

not related to this PR but still - `libsensors-spi` should be a separate component with different HAL implementations (or drivers should use common SPI API if it would ever be available), not a c file withing the same dir with hard-coded zynq API.

Reply:
Fixed the comment on reinstalling.

About the libsensors.spi - yeah I am aware of it. As for now I have created the issue:
 - https://github.com/phoenix-rtos/phoenix-rtos-project/issues/1289

But I do not plan on solving this within this PR for now.